src/editor/adapters/change-signature-webview/change-signature.html

```diff
@@ -0,0 +1,137 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <style>
+      table {
+        font-family: arial, sans-serif;
+        border-collapse: collapse;
+      }
+
+      td,
+      th {
+        border: 1px solid #dddddd;
+        text-align: left;
+        padding: 8px;
+      }
+
+      th:last-child {
+        border-top-color: transparent;
+        border-right-color: transparent;
+      }
+
+      .up,
+      .down {
+        cursor: pointer;
+        display: inline-block;
+        width: 8px;
+        margin: 0 0.7rem;
+        font-size: 1.2rem;
+      }
+
+      .up:after {
+        content: "▲";
+      }
+
+      .up:hover:after {
+        color: #625e5e;
+      }
+
+      .down:after {
+        content: "▼";
+      }
+
+      .down:hover:after {
+        color: #625e5e;
+      }
+
+      button {
+        border: 1px solid transparent;
+        border-radius: 5px;
+        line-height: 1.25rem;
+        outline: none;
+        text-align: center;
+        white-space: nowrap;
+        display: inline-block;
+        text-decoration: none;
+        background: #1a85ff;
+        padding: 4px;
+        color: white;
+        font-size: 14px;
+      }
+
+      button:hover {
+        cursor: pointer;
+        color: white;
+      }
```

Comment:
👍 I like that we got the proper syntax highlighting back for the template

It will also make it easy to "design" later